src/test/fuzz/orphanage.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <net_processing.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/setup_common.h>
+#include <txorphanage.h>
+
+void initialize_orphanage()
+{
+    static const auto testing_setup = MakeNoLogFileContext();
+}
+
+FUZZ_TARGET_INIT(orphanage, initialize_orphanage)
```

Comment:
```suggestion
FUZZ_TARGET_INIT(txorphanage, initialize_orphanage)
```

Maybe rename the file and target to `txorphanage`?

Reply:
Sure. `orphanage` is not so explicit. But the source file has the same name `txorphanage.cpp`, is `txorphan` better?